### PR TITLE
Get rid of IE hacks: browser.msie

### PR DESCRIPTION
**Pull request by jacksonrr3**
Get rid of `browser.msie` property at core/utils/browser, and delete it where it using.

### CODE REVIEW

**Review by novsstation — DISMISSED** [review]

File: js/core/utils/date.js [hunk 1]
```diff
@@ -626,7 +623,7 @@ const getDatesOfInterval = function(startDate, endDate, step) {
 };
 
 const createDate = function(date) {
```
Comment (nightskylark) [inline js/core/utils/date.js]: Let's get rid of this method. It's not needed anymore.

File: js/core/utils/position.js [hunk 2]
```diff
@@ -10,19 +9,7 @@ const getDefaultAlignment = (isRtlEnabled) => {
 };
 
 const getElementsFromPoint = (x, y) => {
```
Comment (nightskylark) [inline js/core/utils/position.js]: Same here

File: js/renovation/ui/form/_toDelete/ui.form.js [hunk 3]
```diff
@@ -1173,11 +1172,7 @@ const Form = Widget.inherit({
         return this.$element().find('.' + FIELD_ITEM_CONTENT_CLASS + ' [tabindex]').first();
     },
 
-    _visibilityChanged: function(visible) {
-        if(visible && browser.msie) {
-            this._refresh();
-        }
-    },
+    _visibilityChanged: function() {},
```
Comment (nightskylark) [inline js/renovation/ui/form/_toDelete/ui.form.js:1175]: Why did you leave an empty method?

File: js/ui/form/ui.form.js [hunk 4]
```diff
@@ -1178,11 +1177,8 @@ const Form = Widget.inherit({
         return this.$element().find('.' + FIELD_ITEM_CONTENT_CLASS + ' [tabindex]').first();
     },
 
-    _visibilityChanged: function(visible) {
-        if(visible && browser.msie) {
-            this._refresh();
-        }
-    },
+    // TODO. Deleting this method causes the DataGrid.editing test error.
```
Comment (nightskylark) [inline js/ui/form/ui.form.js]: Let's not leave todos in PRs. If we need to solve some issue, let's do it now. Nobody will return to it later.

**Review by nightskylark — CHANGES_REQUESTED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by nightskylark — DISMISSED** [review]

**Review by nightskylark — APPROVED** [review]

**Pull request merged**